packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/callbacks/ViewCallbackDelegate.kt

```diff
@@ -0,0 +1,27 @@
+@file:OptIn(ExperimentalStdlibApi::class)
+
+package expo.modules.kotlin.callbacks
+
+import android.view.View
+import java.lang.ref.WeakReference
+import kotlin.reflect.KProperty
+import kotlin.reflect.KType
+import kotlin.reflect.typeOf
+
+class ViewCallbackDelegate<T>(private val type: KType, view: View) {
+  private val viewHolder = WeakReference(view)
+  internal var wasValidated = false
```

Comment:
If it `was` validated then it's currently is `validated` state right? So let's make it simpler:
```suggestion
  internal var isValidated = false
```